cogs/render_codeblock.py

```diff
@@ -132,9 +125,9 @@ def extract_manim_snippets(msg) -> None | str:
     pattern = re.compile(r"```(?:py|python)?([^`]*def construct[^`]*)```")
     return pattern.findall(msg)
 
-def render_animation_snippet(code_message, cli_flags=None) -> discord.File:
-    if cli_flags is None:
-        cli_flags = ""
+
+async def render_animation_snippet(code_message, cli_flags=[]) -> Dict[str, Any]:
+    dockerclient = aiodocker.Docker()
```

Comment:
Complex data types as default arguments do bother me a bit, I'd rather write it like this:

```suggestion
async def render_animation_snippet(code_message, cli_flags=None) -> Dict[str, Any]:
    if cli_flags is None:
        cli_flags = []

    dockerclient = aiodocker.Docker()
```